huaweicloud/services/eip/resource_huaweicloud_eip_associate.go

```diff
@@ -37,18 +42,41 @@ func ResourceEIPAssociate() *schema.Resource {
 				Computed: true,
 				ForceNew: true,
 			},
-
-			"port_id": {
+			"public_ip": {
 				Type:     schema.TypeString,
 				Required: true,
 				ForceNew: true,
 			},
-
-			"public_ip": {
+			"port_id": {
```

Comment:
Suggest to add Conflicts between `port_id` and `fixed_ip`.

Reply:
+1

Reply:
**ExactlyOneOf**  is updated